bin/reth/src/config.rs

```diff
@@ -38,6 +39,7 @@ impl Config {
             .peer_config(peer_config)
             .genesis_hash(genesis_hash)
             .chain_id(chain_id)
+            .discovery(Discv4Config::builder().external_ip_resolver(Some(nat_resolution_method)))
```

Comment:
I resolved the missing import but external_ip_resolver() returns &mut Discv4ConfigBuilder not Discv4ConfigBuilder. Not sure, how best to handle this.

Reply:
Resovled it.